example-projects/nats-example/src/main.rs

```diff
@@ -0,0 +1,50 @@
+use std::{error::Error, thread};
+
+use cloudevents::binding::nats::{MessageExt, NatsCloudEvent};
+use cloudevents::{EventBuilder, EventBuilderV10, Event};
+use serde_json::json;
+
+/// First spin up a nats server i.e. 
+/// ```bash
+/// docker run -p 4222:4222 -ti nats:latest
```

Comment:
is `latest` a good idea or should we point to an exact version here?

Reply:
I got that from [Nats Quick Start](https://docs.nats.io/running-a-nats-service/introduction/installation) and I guess it's safe to run these example code with latest version of Nats Server.

Reply:
I think it's fine to stick to the `latest`.

We usually advise our users to be up to date with latest released `nats-server` version.

Even in the case of NATS 3.0 anywhere in the future, I assume this version of the client will bind to the current protocol compatibility version when talking to the server.